src/Quidjibo.DataProtection.Tests/Protectors/AesPayloadProtectorTests.cs

```diff
@@ -49,11 +49,16 @@ public async Task ShouldVerifyTest()
 
             var protectedPayload = await _sut.ProtectAsync(payload, CancellationToken.None);
 
-            protectedPayload[49] = (byte)(protectedPayload[49] ^ 255);
+            for(var i = 1; i < protectedPayload.Length; i++)
+            {
+                var payloadCopy = new byte[protectedPayload.Length];
+                protectedPayload.AsSpan().CopyTo(payloadCopy);
+                payloadCopy[i] ^= 255;
```

Comment:
This test was changed to verify that _any_ change to the payload, exception the version, causes a MAC exception.